modin/logging/config.py

```diff
@@ -83,14 +86,17 @@ def bytes_int_to_str(num_bytes, suffix="B"):
         Human-readable string format.
     """
     factor = 1000
+    bytes: float = num_bytes
     for unit in ["", "K", "M", "G", "T", "P"]:
-        if num_bytes < factor:
-            return f"{num_bytes:.2f}{unit}{suffix}"
-        num_bytes /= factor
-    return f"{num_bytes * 1000:.2f}P{suffix}"
+        if bytes < factor:
+            return f"{bytes:.2f}{unit}{suffix}"
+        bytes /= factor
+    return f"{bytes * 1000:.2f}P{suffix}"
 
 
-def _create_logger(namespace, job_id, log_name, log_level):
+def _create_logger(
+    namespace: str, job_id: str, log_name: str, log_level: int
```

Comment:
We would need to change some of the code to go from using the enum to a string. Is that a change we want to make?

Reply:
I think we would also have to reflect the changes on all the callers of the function. Is that something. we want to do as well?

Reply:
Yes, but no need to do any of this in this PR, this was more of a general comment. If you think it's worth changing, feel free to open an issue for it.

Reply:
Yeah looking at it now, I think that we should probably unify the log_levels to either be `str` or `int` or maybe even both. I'll create an issue for this. It might be a good one for new folks to try working on.